MonstieWash/Assets/Scripts/UI/CustomPolaroidLoader.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.IO;
+
+public class CustomPolaroidLoader : MonoBehaviour
+{
+    [SerializeField] private string saveLocation; //Should match exactly with decoration manager set save path.
+
+    private Image img;
+
+    private void Awake()
+    {
+        img = GetComponent<Image>();
+
+        //Get file location lenght
+        var fileCount = Directory.GetFiles(Application.persistentDataPath).Length - 1;
+
+
+        Texture2D spriteTex = LoadTexture(Application.persistentDataPath + saveLocation + fileCount.ToString() + ".Png");
+        Sprite NewSprite = Sprite.Create(spriteTex, new Rect(0, 0, spriteTex.width, spriteTex.height), new Vector2(0, 0), 100f, 0, SpriteMeshType.Tight);
```

Comment:
### 🗒️ **Convention** | _This is for maintaining our coding conventions._

These can be using `var` too. Also gotta fix the case on `NewSprite`.

```suggestion
        var spriteTex = LoadTexture(Application.persistentDataPath + saveLocation + fileCount.ToString() + ".Png");
        var newSprite = Sprite.Create(spriteTex, new Rect(0, 0, spriteTex.width, spriteTex.height), new Vector2(0, 0), 100f, 0, SpriteMeshType.Tight);
```